crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -99,16 +99,23 @@ def health_check
       end
 
       def ceph_status
-        {}.tap do |ret|
-          ceph_node = ::Node.find("roles:ceph-mon AND ceph_config_environment:*").first
-          next unless ceph_node
-          ssh_retval = ceph_node.run_ssh_cmd("LANG=C ceph health 2>&1")
-          unless ssh_retval[:stdout].include? "HEALTH_OK"
-            ret["errors"] = [
-              "ceph cluster health check failed with #{ssh_retval[:stdout]}"
-            ]
-          end
+        ret = {}
+        ceph_node = ::Node.find("roles:ceph-mon AND ceph_config_environment:*").first
+        return ret if ceph_node.nil?
+
+        ssh_retval = ceph_node.run_ssh_cmd("LANG=C ceph health 2>&1")
+        unless ssh_retval[:stdout].include? "HEALTH_OK"
```

Comment:
I have once concern about this (which needn't stop this PR going in, it can be a separate issue if you like) -- if I have a cluster with too many (or two few) PGs per OSD, that is otherwise functional, `ceph health` will print something like `HEALTH_WARN too many PGs per OSD (624 > max 300)`. That's a condition that shouldn't prevent an upgrade IMO, because the cluster is otherwise functional, and it's not necessarily possible to even fix that case without destroying pools or adding more OSDs.